openhtf/plugs/README.md

```diff
@@ -0,0 +1,10 @@
+# Usefull thinks!
```

Comment:
Not sure what was meant here, but let's take it out or change it to something more descriptive.